content/docs/studio/user-guide/model-registry/use-models.md

```diff
@@ -22,12 +24,13 @@ You can also construct this command manually using this template:
 $ dvc get ${GIT_REPO} ${MODEL_PATH} --rev ${GIT_REV}
 ```
 
-Let's download the model in the
```

Comment:
@aguschin The `seg-model` model does not have an entry in `dvc.yaml` in the repo. And it does not show up in the project or MR if I add this repo in Studio. So, this example is confusing. Can we use an example that is already available in the demo model registry?

Reply:
Right. I need to fix that. @daavoo, did you complete updating example-get-started already? I guess I need to add some GTO command calls in generate.sh.

Reply:
`example-get-started-experiments` , yes, please add to `generate.sh` so we can keep updating on top without worrying

Reply:
@tapadipti - I made a commit to address these changes.

Reply:
@aguschin There is now a `pool-segmentation` model available, right? Why change this to use the other repo?